proposals/csharp-10.0/parameterless-struct-constructors.md

```diff
@@ -0,0 +1,157 @@
+# Parameterless struct constructors
+
+## Summary
+
+Support parameterless constructors and instance field initializers for struct types.
+
+## Proposal
+
+### Instance field initializers
+Instance field declarations for a struct may include initializers.
+
+As with [class field initializers](https://github.com/dotnet/csharplang/blob/master/spec/classes.md#instance-field-initialization):
+> A variable initializer for an instance field cannot reference the instance being created. 
+
+### Constructors
+A struct may declare a parameterless instance constructor.
+The struct can be declared as `struct`, `readonly struct`, or `ref struct`.
+
+If the struct does not declare a parameterless instance constructor, and the struct has no fields with variable initializers, the struct (see [struct constructors](https://github.com/dotnet/csharplang/blob/master/spec/structs.md#constructors)) ...
+> implicitly has a parameterless instance constructor which always returns the value that results from setting all value type fields to their default value and all reference type fields to null.
+
+If the struct does not declare a parameterless instance constructor, and the struct has field initializers, a `public` parameterless instance constructor is synthesized.
+The parameterless constructor is synthesized even if all initializer values are zeros.
+
+### Modifiers
+A parameterless instance constructor may be less accessible than the struct.
```

Comment:
```suggestion
A parameterless instance constructor may be less accessible than the containing `struct`.
```

Reply:
Might want to add that this has consequencs for otherr use cases that will be detailed later.